test/int64.js

```diff
@@ -0,0 +1,529 @@
+    },
+    {
+      value: {},
+      expectedString: '0',
+      message: 'must create zero Int64 from an invalid constructor argument',
+    },
+    {
+      value: NaN,
+      expectedString: '0',
+      message: 'must create zero Int64 from a NaN constructor argument',
+    },
+    {
+      value: undefined,
+      expectedString: '0',
+      message: 'must create zero Int64 from an undefined constructor argument',
+    },
+    {
+      value: '0xinvalidnumber',
+      expectedString: '0',
+      message:
+        'must create zero Int64 from an invalid hex string',
+    },
+    {
+      value: '0o999',
+      expectedString: '0',
+      message:
+        'must create zero Int64 from an invalid oct string',
+    },
+    {
+      value: '0b333',
+      expectedString: '0',
+      message:
+        'must create zero Int64 from an invalid bin string',
+    },
+    {
+      value: 9999999999999999999999999999999999,
+      expectedString: '0',
+      message:
+        'must create zero Int64 from an unsafe integer number',
+    },
+    {
+      value: '0xffffffffffffffff',
+      expectedString: '18446744073709551615',
+      message: 'must create Int64 from a big hex number provided as a string',
+    },
+    {
+      value:
+        '0b1010101010101010101010101010101010101010101010101010101010101010',
+      expectedString: '12297829382473034410',
+      message: 'must create Int64 from a big bin number provided as a string',
+    },
+    {
+      value: '0o1777777777777777777777',
+      expectedString: '18446744073709551615',
+      message: 'must create Int64 from a big oct number provided as a string',
+    },
+    {
+      value: '18446744073709551615',
+      expectedString: '18446744073709551615',
+      message: 'must create Int64 from a big dec number provided as a string',
+    },
+  ].forEach(testcase => {
+    const Int64 = new common.Int64(testcase.value);
+    test.strictSame(
+      Int64.toString(),
+      testcase.expectedString,
+      testcase.message
+    );
+  });
+
+  test.end();
+});
+
+metatests.test('Int64 division by zero', test => {
+  const nonZeroNumber = new common.Int64(10);
+  const zero = new common.Int64(0);
+  test.throws(() => {
+    common.Int64.div(nonZeroNumber, zero);
+  }, new RangeError('Int64: division by zero'),
+  'must throw when dividing by zero');
+
+  test.end();
+});
+
+metatests.test('Int64.prototype.toString() with invalid radix', test => {
+  const number = new common.Int64(10);
+  const errorExpected =
+    new RangeError('toString() radix argument must be between 2 and 36');
+
+  test.throws(() => {
+    number.toString(1000);
+  }, errorExpected, 'must throw when toString() radix argument is too big');
+  test.throws(() => {
+    number.toString(0);
+  }, errorExpected, 'must throw when toString() radix argument is too small');
+  test.throws(() => {
+    number.toString(-1);
+  }, errorExpected, 'must throw when toString() radix argument is negative');
+
+  test.end();
+});
+*/
```

Comment:
Why do we have 162 commented lines here?

Reply:
That's the tests that will be fixed in the next commit.